video/video.ino

```diff
@@ -335,6 +335,15 @@ bool processTerminal() {
 				if (strcmp("S!", seq) == 0) {
 					suspendTerminal();
 				}
+				if (seq[0] == 'F') {
+					uint32_t fontnum = textToInt32(seq + 1);
+					if (fontnum >= 0) {
+						auto font = fonts[fontnum];  // get shared_ptr to font -- was fonts[bufferID]
+						if (font != nullptr) {  // check it's defined
+						Terminal->loadFont(font.get());

```

Comment:
for safety, and potential future-proofing, I suspect that we should only call `loadFont` if the font is mono-spaced.  I think that means checking that `font->chptr == nullptr` to make sure there's no spacing info

Reply:
Good plan - that looks like the same test used in getCharPtr.